Library/Homebrew/utils/curl.rb

```
@@ -1,42 +1,46 @@
 require "pathname"
 require "open3"
 
-def curl_args(options = {})
+def curl_executable
   curl = Pathname.new ENV["HOMEBREW_CURL"]
   curl = Pathname.new "/usr/bin/curl" unless curl.exist?
-  raise "#{curl} is not executable" unless curl.exist? && curl.executable?
+  return curl if curl.executable?
+  raise "#{curl} is not executable"
+end
 
+def curl_args(*extra_args, show_output: false, user_agent: :default)
   args = [
-    curl.to_s,
-    "--remote-time",
-    "--location",
+    curl_executable.to_s,
+    "--fail",
```

We don't want this to fail silently when output is requested.